test/test_cohort_defaults.py

```diff
@@ -32,9 +32,9 @@
 
 FILE_FORMAT_1 = "patient_format1_%s.vcf"
 
-def test_default_filter_fn():
+def test_default_filter_fn(summary_fn=variant_count):
```

Comment:
To test this I ran nosetests with the decorator `@nottest` in front of that function & without the decorator. 35 tests ran with the decorator, whereas without the decorator 36 tests ran. So I think it does run. LMK if you think this test should be done differently.